### DISCUSSION TITLE
Welcome to ResXResourceManager Discussions!

### DISCUSSION THREAD

**Discussion by tom-englert** [dotnet/ResXResourceManager#443] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by renishnair-viper** [comment]
Hello 
Is it possible to reuse an existing key in any new sentence?
e.g.
1) Key => ZoomIn 
English => Zoom in 
German => Hineinzoomen

2) Key => ExpandMap
English => We can use the Zoom in feature to expand the map.

So here in 2nd English text can we use the ZoomIn key with in the sentence? 
i.e. We can use the [Key1] feature to expand the map.

**Reply by tom-englert** [reply]
Isn't this rather related to how text is retrieved at runtime? Then RXRM is not involved.
